habitat_baselines/agents/ppo_agents.py

```diff
@@ -81,7 +81,7 @@ def __init__(self, config: Config) -> None:
             observation_space=observation_spaces,
             action_space=action_spaces,
             hidden_size=self.hidden_size,
-            normalize_visual_inputs="rgb" in spaces,
+            # normalize_visual_inputs="rgb" in spaces,
```

Comment:
do we need this commented? Can you add a comment for why we comment it in gala_kinematic?

Reply:
This is a random old file, mypy was complaining about it since the call signature changed.